packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -597,21 +597,16 @@ export async function downloadDocsAndUpsertFHIR({
           const FHIRDocRef = toFHIRDocRef(doc.id, docWithFile, patient);
 
           if (shouldUpsertFHIR) {
-            const [fhir, search] = await Promise.allSettled([
-              upsertDocumentToFHIRServer(cxId, FHIRDocRef).catch(error => {
+            const [fhir] = await Promise.allSettled([
+              upsertDocumentToFHIRServer(cxId, FHIRDocRef, log).catch(error => {
                 const context = "upsertDocumentToFHIRServer";
                 reportFHIRError({ patientId: patient.id, doc, error, context, log });
                 errorReported = true;
                 throw error;
               }),
-              ingestIntoSearchEngine(patient, FHIRDocRef, file, requestId).catch(error => {
-                const context = "ingestIntoSearchEngine";
-                reportFHIRError({ patientId: patient.id, doc, error, context, log });
-                errorReported = true;
-                throw error;
-              }),
+              ingestIntoSearchEngine(patient, FHIRDocRef, file, requestId, log),
```

Comment:
Intentionally making the search ingestion to fail gracefully so it doesn't impact the download progress for cxs. The function has its own error handling code notifying Sentry.